attiny/attiny.ino

```diff
@@ -248,10 +247,12 @@ void initTimer1() {
 
 ISR(TIMER1_COMPA_vect) {
   if(minuteCountdown <= 0) {
-    minuteCountdown = 120; // =60/0.5 because interrupt every 0.5 seconds
+    minuteCountdown = MINUTE_COUNTDOWN;
+    cli();
```

Comment:
The cli and sei aren't necessary here because this is an interrupt handler so interrupts are already disabled.